WordPress/src/main/java/org/wordpress/android/ui/bloggingreminders/BloggingRemindersViewModel.kt

```diff
@@ -167,7 +167,7 @@ class BloggingRemindersViewModel @Inject constructor(
                             bloggingRemindersModel.hour,
                             bloggingRemindersModel.minute,
                             bloggingRemindersModel.toReminderConfig())
-                    analyticsTracker.trackRemindersScheduled(daysCount)
+                    analyticsTracker.trackRemindersScheduled(daysCount, bloggingRemindersModel.getNotificationTime())
```

Comment:
This is going to be formatted according to the user's locale and would result in different values for the same time, which could make it harder for us to gather the data accurately.

Perhaps we use a standard 24-hour formatter for tracking time, like `DateTimeFormatter.ofPattern("HH:mm")`. Let me know what you think.